File: ATB_1.1_src/src/main/java/edu/stanford/isis/atb/domain/template/AnatomicEntityCharacteristic.java
/*
*  Copyright Northwestern University
*  Copyright Stanford University (ATB 1.0 and ATS 1.0)
*
*  Distributed under the OSI-approved BSD 3-Clause License.
*  See http://ncip.github.com/annotation-and-image-markup/LICENSE.txt for details.
*/



package edu.stanford.isis.atb.domain.template;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Root;

/**
 * @author Vitaliy Semeshko
 */

@Root(name="AnatomicEntityCharacteristic", strict=false)
public class AnatomicEntityCharacteristic extends ComponentCharacteristic {

	@Attribute
	private boolean annotatorConfidence;
	
	public boolean isAnnotatorConfidence() {
		return annotatorConfidence;
	}

	public void setAnnotatorConfidence(boolean annotatorConfidence) {
		this.annotatorConfidence = annotatorConfidence;
	}

	@Override
	public String toString() {
		return "Anatomic Entity Characteristic: " + getLabel();
	}
	
}

File: AimPlugin3.0.4/AIM.Annotation/Template/AnatomicEntityCharacteristic.cs
#region License

//L
// 2007 - 2013 Copyright Northwestern University
//
// Distributed under the OSI-approved BSD 3-Clause License.
// See http://ncip.github.com/annotation-and-image-markup/LICENSE.txt for details.
//L

#endregion

using System.Xml.Serialization;

namespace AIM.Annotation.Template
{
	public class AnatomicEntityCharacteristic : ComponentCharacteristic
	{
		[XmlAttribute("annotatorConfidence")]
		public bool AnnotatorConfidence { get; set; }

		[XmlElement("QuestionType")]
		public ValidTerm QuestionType { get; set; }
	}
}
